### ISSUE TITLE
Create New- User. Add new license error. 

### ISSUE THREAD

**Issue by ccfcs-ttu** [Tizra/Tizra-Customer-Tracker#407]
When using the create new- user function and entering an email that already exists in our customer database the user profile for said email is opened. When then trying to add a new license we are getting an "error unknown...java...error in class" message. We do not get this message when doing the long version of users> search> choosing and opening the user profile page and then adding a new license.

**Comment by daviddurand** [comment]
We'll look into this, but in the meantime, it seems that there's a viable workaround.

**Comment by ccfcs-ttu** [comment]
Yeah, the workaround works fine and this isn’t necessarily a high priority for us. I just wanted to make you aware of it.


Thanks,
                             Derrick Pedersen
                          IT Support Technician I
                                  (806) 834-0845
[Untitled]

**Comment by daviddurand** [comment]
Great, just wanted to be sure I wasn't missing anything. Pretty sure we'll get to it fairly soon in any case.
